index.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Capstone</title>
+		<link
+			rel="stylesheet"
+			href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+		/>
+		<link rel="stylesheet" href="dist/css/styles.css" />
+	</head>
+	<body class="dark">
+		<header>
+			<div class="top-header">
+				<ul class="">
+					<li>
+						<a href=""
+							><img
+								src="assets/images/icons/icon-facebook.svg"
+								alt="facebook-icon"
+						/></a>
+					</li>
+					<li>
+						<a href=""
+							><img
+								src="assets/images/icons/icon-twitter.svg"
+								alt="twitter-icon"
+						/></a>
+					</li>
+					<li><a href="">English</a></li>
+					<li><a href="">My Page</a></li>
+					<li><a href="">Logout</a></li>
+				</ul>
+			</div>
+			<nav>
+				<div class="logo-container hide">
+					<a href="#"
+						><img src="assets/images/icons/icons8-bitcoin.gif" alt="logo" />
+						<div class="logo-title">
+							<span>CryptoFest </span>
+							<span>Global Conference</span>
+						</div></a
+					>
+				</div>
+				<div class="mobile-menu">
+					<img
+						src="assets/images/icons/button.svg"
+						alt="hamburger image"
+						class="toggle hide"
+					/>
+				</div>
+				<div class="desktop-menu limited">
+					<ul>
+						<li><a href="./about.html">About</a></li>
+						<li><a href="#">Program</a></li>
+						<li class="btn"><a href="#">Join</a></li>
+						<li><a href="">Sponsor</a></li>
+						<li><a href="">News</a></li>
+						<li><a href="">Crypto Summit</a></li>
+					</ul>
+				</div>
+			</nav>
+		</header>
+		<main>
+			<section class="hero">
+				<div class="headline"></div>
+				<div class="headline-details">
+					<p>Calling on all Crypto Enthusiasts</p>
+					<h1>CRYPTO FEST VIRTUAL <br />CONFERENCE <br />2021</h1>
+					<span>Monday <span class="stroke"></span>February 28, 2022</span>
+					<p>CRYPTO MINING &#38;  ITS EFFECTS</p>
```

Comment:
- [ ] Date and conference subject should have a bigger text size with some boldness, please adjust your styling to match the Reference Design.